api/src/main/java/eu/coatrack/api/ProxyStates.java

```diff
@@ -0,0 +1,40 @@
+package eu.coatrack.api;
+
+/*-
+ * #%L
+ * coatrack-api
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+public enum ProxyStates {
+
+    CRITICAL("critical-state"),
+    WARNING("warning-state"),
+    OK("ok-state"),
+    IGNORE("ignore-state"),
+    NEVER_CONNECTED("");
+
+    private final String cssClassForGui;
```

Comment:
I think that this class should not know anything about the frontend. For example it should not care about being part of a web application or a client agent.

Reply:
So here it was a suggestion from Gunnar because of the name displayShortString.
So as this string was not displayed on the GUI, but on a CSS class, he defined to change the name to cssClassForGui and I accepted as it is easier to understand.

Reply:
Done, so now everything related to the Gateway status css classes is on Frontend.